R/pkg/DESCRIPTION

```diff
@@ -54,5 +54,5 @@ Collate:
     'types.R'
     'utils.R'
     'window.R'
-RoxygenNote: 5.0.1
+RoxygenNote: 6.0.1
```

Comment:
Yea, it'd update this line automatically, if you have R and Roxygen2 is a different version. 
6.0 was just released 2 weeks ago and it seems not at all everything is backward compatible, so probably safer for now to hold on (that and might/might not match what we have on Jenkins)